fn_grpc_interface/MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include README.md
 include fn_grpc_interface/util/*
 include fn_grpc_interface/LICENSE
-include fn_grpc_interface/doc/*
\ No newline at end of file
+include fn_grpc_interface/screenshots
```

Comment:
`screenshots` is at an upper level in the directory so it should be:
```
include screenshots/*
```

Reply:
ok